Release Notes/2019/2019-06-11-cloud-platform-release-notes.md

```diff
@@ -23,7 +23,7 @@ On May 14, 2019, Microsoft announced CVE-2019-0708, a critical flaw in Remote De
 
 If you have Remote Desktop Services enabled and a public IP, you are at a high risk of being compromised and need to update all machines running Windows 2008 or older ASAP. At the very least, please disable Remote Desktop Services if you can until the updates are performed.  
 
-In all cases, Lumen recommends that you to install the updates for this vulnerability as soon as possible.  Where able, migrate applications to a newer version of Windows as Windows 2008 is going end of support Jan 2020.  For responses to FAQs regarding Windows 2008 support please see [this Knowledge Base article](../../Support/windows-2008-end-of-vendor-support-faq.md).
+In all cases, Lumen recommends that you to install the updates for this vulnerability as soon as possible.  Where able, migrate applications to a newer version of Windows as Windows 2008 is going end of support Jan 2020.  For responses to FAQs regarding Windows 2008 support please see [this Knowledge Base article](../../support/windows-2008-end-of-vendor-support-faq.md).
```

Comment:
Same here. The support folder is capitalized so this will need to be as well.

Reply:
I changed "support" to "Support."